packages/vitest/src/node/reporters/html.ts

```diff
@@ -0,0 +1,78 @@
+import { existsSync, promises as fs } from 'fs'
```

Comment:
Reporter should be part of @vitest/ui package.

Reply:
It mean `@vitest/ui/reporter` or copy the ui dist from ui lib?

Reply:
I mean this file should be inside `@vitest/ui` package. We can add custom logic for html reporter to `createReporters` utility